rotkehlchen/db/schema.py

```diff
@@ -328,6 +328,17 @@
 );
 """
 
+DB_CREATE_EVM_TRANSACTIONS_AUTHORIZATION = """
+CREATE TABLE IF NOT EXISTS evm_transactions_authorization (
```

Comment:
Just like transactions is plural

```suggestion
DB_CREATE_EVM_TRANSACTION_AUTHORIZATIONS = """
CREATE TABLE IF NOT EXISTS evm_transaction_authorizations (
```